bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/connection/Clip2Bridge.java

```diff
@@ -693,23 +790,31 @@ public Resources getResources(ResourceReference reference) throws ApiException,
      */
     private Resources getResourcesImpl(ResourceReference reference)
             throws HttpUnauthorizedException, ApiException, InterruptedException {
-        Session session = http2Session;
-        if (Objects.isNull(session) || session.isClosed()) {
-            throw new ApiException("HTTP 2 session is null or closed");
+        // temporary work around for issue #15468
+        if (reference.getType() == ResourceType.ERROR) {
+            LOGGER.warn("Resource '{}' type unknown (issue #15468) => GET aborted", reference.getId());
```

Comment:
I certainly see the `taurus_7455` resources in the JSON from my Hue bridge. I hope it is not a unique issue to me (or I have been hacked). Perhaps you can turn on trace and restart your bridge thing to check if you see it too?

Reply:
I see the warning. Should I check payload to confirm?

Reply:
Yes it would be good to see your payload.

Reply:
I see exactly the same as you have described in #15468. Most of my light devices are non-Hue, but for two specific Hue bulbs I see:

```json
                {
                    "rid": "00000000-0000-0000-0000-000000000000",
                    "rtype": "taurus_7455"
                }
```

And I'm now on 1.59.1959194030.

We could consider adding `TAURUS_7455` to the `ResourceType` enum in order to recognize and ignore this, but still log other unknown types as warnings.

Or perhaps add a member variable for storing the name in case of `ERROR` here (line 66):

https://github.com/openhab/openhab-addons/blob/d36833feef860a068b782a6d4b5ccccdb8bf2e53/bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/dto/clip2/enums/ResourceType.java#L61-L70

It would then also be possible to log the unknown type (i.e. in this case "taurus_7455"). And we wouldn't need to add `TAURUS_7455` a enum member, since we could check the name in case of `ERROR`.

WDYT?

Reply:
>WDYT?

I am half with you, and half not. I think that 'taurus_7455' is so obviously not an intentional design function, but rather a developer aberration, that is likely to dissappear in the next firmware release. So I am inclined not to hard code a solution to a problem that will quite likely not exist on future. But then again I may be wrong. I suggest to leave  the code as it is currently, but perhaps mark the OH 'taurus_7455' issue as NOT closed, so we can continue to track it when next firmware is released.

Reply:
If we are leaving it for now hoping that the firmware will soon be fixed, we should probably to back to logging as warning, so that it will be reported if new (real) unknown resource types will appear?

What do you think about my second proposal of improving `ResourceType` to make it possible to log the actual unknown resource type? This would give some more immediate relevant information.

Reply:
> proposal of improving ResourceType to make it possible to log the actual unknown resource type

We could do that. But the logging would have to be done in the handler when the resource is discovered rather than in Clip2Bridge when we try to GET it. 

My reason for catching it in Clip2Bridge is to keep the scope of this PR focussed on just this one class.

Reply:
I'm fine scoping this PR to not deal with `taurus_7455`. I have added a small comment for the logging if we want to be consistent with this decision. It's a small thing, and I really want to hit the "merge" button, but let's first get this right.

Reply:
Ok. I just made a commit which should cover it.